.github/workflows/lint.yml

```diff
@@ -22,3 +22,9 @@ jobs:
     - run: black --check *.py */
     - run: isort --check-only *.py */
     - run: flake8
+    - run: pip install scspell3k
```

Comment:
This line shouldn't be needed as it's in req_dev and that is installed a few lines up?